spec/acceptance/docker_full_spec.rb

```diff
@@ -0,0 +1,643 @@
+require 'spec_helper_acceptance'
+
+describe 'Temp OS hacks' do
```

Comment:
These tests are written for the 3 supported OS.

We only need to install docker manually to enable the extras repo for RHEL.